index.js

```diff
@@ -9,14 +9,13 @@ module.exports = {
   included: function(app) {
     this._super.included(app);
     if (process.env.EMBER_CLI_FASTBOOT !== 'true') {
+      // In nested addons, app.bowerDirectory might not be available
+      var bowerDirectory = app.bowerDirectory || 'bower_components';
+      // In ember-cli < 2.7, this.import is not available, so fall back to use app.import
+      var importShim = typeof this.import !== 'undefined' ? this : app;
```

Comment:
As far as I understand, `this.import` is supposed to work in arbitrary depth? Of course, as mentioned in #10, if you want to use this in your nested addon/engine, you will be required to use either ember-cli >= 2.7. or install https://github.com/ember-cli/ember-cli-import-polyfill.